server/src/helpers/admin.js

```diff
@@ -8,15 +8,26 @@ const generateAdminPassword = () => {
     if (err) throw err
     console.log('generate-admin-password >> file saved!')
   })
-  return id
+  return id.toString()
 }
 
 const getAdminPassword = () => {
   const token = fs.readFileSync('./src/config/admin-password.txt')
   return token.toString()
 }
 
+const getCreateAdminPassword = () => {
+  const password = getAdminPassword()
+  console.log('gotAdminPassword', password)  
```

Comment:
on the server use the debug(...) over console.log(..)
always prefix the string content with something that ids the module. In this case you might use 'Admin getCreateAdminPassword .... remaining text'